mmdet/models/backbones/hrnet.py

```diff
@@ -0,0 +1,536 @@
+import logging
+
+import torch.nn as nn
+from mmcv.cnn import constant_init, kaiming_init
+from mmcv.runner import load_checkpoint
+
+from ..registry import BACKBONES
+from ..utils import build_norm_layer
+
+
+class BasicBlock(nn.Module):
```

Comment:
It seems that `BasicBlock` and `Bottleneck` are the same as in ResNet. You may just import them from the resnet module.